src/opera/util/render_jinja2.py

```diff
@@ -155,10 +177,17 @@ def render_jinja2(template_filename: str, input_data: dict, logger: PgeLogger =
 
     rendered_text = template.render(input_data)
 
+    if validator is not None:
+        try:
+            validator(rendered_text)
+        except Exception as err:
+            raise RuntimeError(f"Error rendering Jinja2 template: {err}") from err
```

Comment:
Lets change this to "Error parsing rendered ISO XML" so its a little more obvious that the failure is during parsing, not creation of the XML from the template.